packages/list-reusable-blocks/src/components/import-dropdown/style.scss

```diff
@@ -1,3 +1,8 @@
 .list-reusable-blocks-import-dropdown__content .components-popover__content {
 	padding: 10px;
 }
+
+[class].list-reusable-blocks-import-dropdown__button {
```

Comment:
`[class]` selector added for more specificity.

Reply:
It's a bummer this is necessary, but it makes sense since the other button is outside our control.